source/Minable.cpp

```diff
@@ -201,3 +201,14 @@ const map<const Outfit *, int> &Minable::Payload() const
 {
 	return payload;
 }
+
+
+
+// Get the value of the asteroid.
+const int64_t &Minable::GetCost()
+{
+	if(!cost)
+		for(auto it : payload)
+			cost += it.first->Cost();
```

Comment:
Some plugins have different items that can spawn from a minable.  For example, midnight plugin has space junk that has a chance to spawn outfits + junk.

Reply:
After some discussion on discord:

* This can't be calculated in Load because the payloads may or may not have been loaded so total cost is not available.
* Payload can potentially be negative.
* Probably best to match other data types and implement a `FinishLoading()` method.

Reply:
https://github.com/endless-sky/endless-sky/pull/6669/commits/d44f549c9d0517bcd2125212bd36feb1e35bf20a addresses this